client/src/components/AdminCard/index.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import { Button } from 'antd';
+import './style.css';
+
+class AdminCard extends Component {
+  state = {
+    students: [],
+  };
+
+  async componentDidMount() {
+    const { studentNames } = this.props;
+    if (studentNames) {
+      const result = await axios.get(
+        `../../../api/v1/projects/${studentNames}/alumni`
```

Comment:
Will be better if you used the same variables names which we used in backend routes
```suggestion
        `/api/v1/projects/${projectId}/alumni`
```